ldap/3.2/modules/ROOT/pages/ldap-connector-reference.adoc

```diff
@@ -735,17 +730,17 @@ reduce the value of the fetch size.
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
 | Base DN a| String |  The base DN of the LDAP search. |  | x
-| Filter a| String |  A valid LDAP filter. The LDAP connector supports LDAP search filters as defined in RFC 2254. |  | x
-| Attributes a| Array of String |  A list of the attributes to return in the result. If the attributes list is empty or null, then by default all LDAP entry attributes are returned. |  |
+| Filter a| String |  A valid LDAP filter. LDAP connector supports LDAP search filters as defined in RFC 2254. |  | x
+| Attributes a| Array of String |  A list of the attributes to return in the result. If the attributes list is empty or null then, by default, all LDAP entry attributes are returned. |  |
 | Scope a| Enumeration, one of:
 
 ** OBJECT
 ** ONE_LEVEL
 ** SUB_TREE |  The scope of the search. Valid attributes are:
 
-* OBJECT: This value is used to indicate searching only the entry at the base DN, resulting in only that entry being returned (keeping in mind that it also has to meet the search filter criteria!)
-* ONE_LEVEL: This value is used to indicate searching all entries one level under the base DN - but not including the base DN and not including any entries under that one level under the base DN.
-* SUB_TREE: This value is used to indicate searching of all entries at all levels under and including the specified base DN. |  ONE_LEVEL |
+* OBJECT: Use to search only the entry at the base DN so that only that entry will be returned. This must meet the search filter criteria.
+* ONE_LEVEL: This value is used for searching all entries one level under the base DN and does not include entries in the base DN or any entries one level under the base DN.
```

Comment:
Suggestion to remove "This value is used for":  "Searches all entries one level..."